app/assets/javascripts/webchat/library.js

```diff
@@ -35,7 +35,7 @@
 
     function handleOpenChat (evt) {
       evt.preventDefault()
-      this.dataset.redirect =="true" ? window.location.href = openUrl : global.open(openUrl, 'newwin', 'width=366,height=516')
+      this.dataset.redirect == "true" ? window.location.href = openUrl : global.open(openUrl, 'newwin', 'width=366,height=516')
```

Comment:
Where is this coming from?

Reply:
from the dom dataset attribute on the a tag

Reply:
you changed it to `payload`, you probably need to change this too